src/main/java/ch/wisv/areafiftylan/users/model/User.java

```diff
@@ -100,6 +101,7 @@ public String getPassword() {
 
     public void resetProfile() {
         this.profile = new Profile();
+        this.profile.setBirthday(LocalDate.now());
```

Comment:
If we reset a profile, should we set the birthdate to now, or to undefined/null ?

It would make more sense to clear it instead of having an arbitrary date filled in.
If that raises errors in the alcohol check, just check for null first.

Reply:
The profile has @nonNull tho

Reply:
I will fill in a arbritrary date

Reply:
Is it not possible for someone to not have a birthdate then? What about the case where ticket transfers occur?

Reply:
I really don't know enough about the ticket transfers to answer this